src/Http/Http.Extensions/test/RequestDelegateGenerator/SharedTypes.cs

```diff
@@ -1,8 +1,29 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
+using Microsoft.AspNetCore.Http.Metadata;
+
 namespace Microsoft.AspNetCore.Http.Generators.Tests;
 
 public class TestService
 {
     public string TestServiceMethod() => "Produced from service!";
 }
+
+public class Todo
+{
+    public int Id { get; set; }
+    public string Name { get; set; } = "Todo";
+    public bool IsComplete { get; set; }
+}
+
+public interface ITodo
```

Comment:
Is this type used? I don't see any references to it.

Reply:
This can be removed for now.